docs/source/examples/managed-jobs.rst

```diff
@@ -546,3 +556,58 @@ The :code:`resources` field has the same spec as a normal SkyPilot job; see `her
   These settings will not take effect if you have an existing controller (either
   stopped or live).  For them to take effect, tear down the existing controller
   first, which requires all in-progress jobs to finish or be canceled.
+
+.. _jobs-controller-sizing:
+
+Best Practices for Scaling Up the Jobs Controller
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tip::
+  For managed jobs, it's highly recommended to use service accounts for cloud authentication. This is so that the jobs controller credentials do not expire.
```

Comment:
Shall we link to https://docs.skypilot.co/en/managed-jobs-docs/cloud-setup/cloud-auth.html?

Reply:
I avoided it because our docs there are pretty poor.

Reply:
I'll add it.